content/en/tracing/trace_explorer/trace_view.md

```diff
@@ -21,23 +21,86 @@ further_reading:
+View an individual [trace][1] to see all of its [spans][2] and associated metadata. Each trace can be visualized as either a Flame Graph, Span List, Waterfall, or Map.
 
-{{< img src="tracing/visualization/trace/service_host_display.png" alt="Service host display" style="width:40%;">}}
+The trace header displays critical trace information, including the root span's service name, resource name, trace ID, end-to-end trace duration, and the trace start datetime. To retain a trace indefinitely, click \"Open Full Page\" and save the URL.
 
-To get a closer look at the flame graph, zoom in by scrolling:
+{{< img src="tracing/trace_view/trace_header.png" alt="Trace header" style="width:90%;">}}
 
-{{< img src="tracing/visualization/trace/trace_zoom.mp4" alt="Trace Error" video="true" width="90%" >}}
+Selected span information is visible at the bottom of the Trace View.
 
-The List view aggregates [resources][4] by [service][3] and sorts them according to their corresponding count of spans. Services are sorted per relative percentage of execution time spent by the trace in each service:
+## Trace visualizations
 
-{{< img src="tracing/visualization/trace/trace_list.png" alt="Trace list" style="width:90%;">}}
+{{< tabs >}}
+{{% tab "Flame Graph" %}}
+
+{{< img src="tracing/trace_view/flamegraph.png" alt="Flame Graph" style="width:90%;">}}
+
+The default visualization. Displays all the spans for an associated trace, color-coded on a timeline. Particularly useful for understanding the execution path of a request and where time was spent over a trace.
+
+To navigate the graph, scroll to zoom, click and drag to move around, and use the minimap to zoom into the selected span or zoom out to the full trace.
+
+The legend details the color coding of the flame graph. Group spans by either **service** (default), **host**, or **container** - and choose to display either **percentage of trace execution time** or **span count** by color grouping. If errors exist on spans in the trace, highlight them in the flame graph by ticking the Errors checkbox under Filter Spans.
+
+{{< img src="tracing/trace_view/flamegraph_legend.mp4" alt="Flame Graph legend" video="true" style="width:90%;">}}
+
+
+{{% /tab %}}
+{{% tab "Span List" %}}
+
+{{< img src="tracing/trace_view/spanlist.png" alt="Trace View" style="width:90%;">}}
+
+Displays [resources][1] by grouping ([service][2] by default) and sorts them according to their count of spans. Particularly useful for scanning latency information by resource or grouping.
+
+Filter resources by type or naming information using the corresponding buttons and text-based search.
+
+{{< img src="tracing/trace_view/spanlist_headers.png" alt="Span List headers" style="width:90%;">}}
+
+Sort groupings by clicking on the corresponding column header: **resource**, **spans**, **average duration**, **execution time**, or **percentage of trace execution time**.
```

Comment:
```suggestion
Click a column heading to sort by that information.
```
I personally don't think we need to explicitly list all of the columns.

Reply:
Did so to avoid conflation with the buttons which exist immediately above them
![image](https://github.com/DataDog/documentation/assets/88278697/b1081ed9-7a7c-426c-93aa-c4812604b8db)
If you're not worried about this, your suggestion is fine by me. If you think the extra clarification is helpful, we could either:
- Have the list of columns (capitalization/grammar matching the UI)
or
- Have a screenshot of the column headers
Up to you!

Reply:
Fair enough! That's a valid point. Let's keep what you have but match the text to how it appears in the UI.